docs/spec/interfaces.md

```diff
@@ -56,23 +55,23 @@ if it previously indicated success.
 A **Sinkable** resource receives events and takes responsibility for further
 delivery. Unlike _Targetable_, a _Sinkable_ cannot return events in its
 response. One example of a _Sinkable_ is a _Channel_ as the target of a
-_Subscription_'s _result_ field.
+_Subscription_'s `reply` field.
 
 <!-- TODO(evankanderson):
 I don't like this example, as it conflates two different things:
 
 That Channel implements Sinkable.
-That Subscription expects a Sinkable in its spec.from.
+That Subscription expects a Sinkable in its spec.reply.
```

Comment:
Not sure which side is correct, but it bears pointing out that this is translating `spec.from` to `spec.reply`. @argent can you clarify original intent?

(This is a comment so it isn't blocking)

Reply:
My reading is that `spec.from` was a typo originally. As the prior section's original example has to do with a `Channel` in the `Subscription`'s `results` and not its `from`.